syntax = "proto3";

import "google/protobuf/empty.proto";

service ESService {
    rpc GetOpenDocument(OpenDocumentRequest) returns (google.protobuf.Empty) {}
}

message OpenDocumentRequest {
    string label = 1;
    repeated string texts = 2;
}